bower.json

```diff
@@ -24,8 +24,11 @@
     "tests"
   ],
   "dependencies": {
-    "bootstrap": "~3.3.6",
-    "ui-bootstrap": "~0.14.3",
+    "angular": "1.X"
+  },
+  "devDependencies": {
+    "bootstrap": "3.X",
```

Comment:
These are not real dependencies they are really dependencies in the demo. Also others still want to use 0.13 and not be locked on 0.14 of ui-bootstrap.. Again along those lines angular-bootstrap has the dist code with a correct bower main definition.